dashboard/src/Entity/Event.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Libero\Dashboard\Entity;
+
+use DateTimeImmutable;
+use DateTimeZone;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table
+ */
+final class Event
+{
+    /**
+     * @ORM\Embedded(class="Aggregate")
+     */
+    private $aggregate;
+
+    /**
+     * @ORM\Column(type="datetime_immutable")
+     */
+    private $dateTime;
+
+    /**
+     * @ORM\Column(type="guid")
+     * @ORM\Id
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $type;
```

Comment:
It's probably worth it to add a basic storage and dump of the `data` field as it may be used to provide debugging information (no schema required)